### PR TITLE
2.9.0 Release notes

### PR DESCRIPTION
**Pull request by svenklemm**

### CODE REVIEW

**Review by iroussos — APPROVED** [review]
Thank you Sven, looks good to me!

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -14,11 +14,76 @@ Want to stay up-to-date with new releases? Subscribe to get
...
+This release includes these noteworthy features:
+* Hierarchical Continuous Aggregates (aka Continuous Aggregate on top of another Continuous Aggregate)
+* Improve `time_bucket_gapfill` function to allow specifying the timezone to bucket
+* Introduce fixed schedules for background jobs and the ability to check job errors.
+* Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.
+
+This release also includes several bug fixes.
+
+**Features**
+* #4476 Batch rows on access node for distributed COPY
```
Comment (charislam) [inline timescaledb/overview/release-notes/index.md:38]:
```suggestion
* #4476 Batch rows on access node for distributed `COPY`
```

File: timescaledb/overview/release-notes/index.md [hunk 2]
```diff
@@ -14,11 +14,76 @@ Want to stay up-to-date with new releases? Subscribe to get
...
+* #4664 Introduce fixed schedules for background jobs
+* #4668 Hierarchical Continuous Aggregates
+* #4670 Add timezone support to time_bucket_gapfill
+* #4678 Add interface for troubleshooting job failures
+* #4718 Add ability to merge chunks while compressing
+* #4786 Extend the now() optimization to also apply to CURRENT_TIMESTAMP
+* #4820 Support parameterized data node scans in joins
+* #4830 Add function to change configuration of a data nodes
+* #4966 Handle DML activity when datanode is not available
+* #4971 Add function to drop stale chunks on a datanode
```
Comment (charislam) [inline timescaledb/overview/release-notes/index.md:50]:
```suggestion
* #4971 Add function to drop stale chunks on a data node
```

File: timescaledb/overview/release-notes/index.md [hunk 3]
```diff
@@ -14,11 +14,76 @@ Want to stay up-to-date with new releases? Subscribe to get
...
+
+**Bugfixes**
```
Comment (charislam) [inline timescaledb/overview/release-notes/index.md:52]:
```suggestion
**Bug fixes**
```

**Review by charislam — APPROVED** [review]
Really minor nits but changing a few of these should also appease the linter enough to pass CI

**Pull request merged**